windows/src/desktop/history.md

```diff
@@ -0,0 +1,359 @@
+# Keyman Desktop Version History
+
+## 10.0 alpha
+* Keyman Desktop moved to open source
+* Support for custom BCP-47 language codes: you can now associate a keyboard with any valid language code in Windows 8 and later
+* Keyman API extensively rewritten and improved
+* Additional user interface language - Turkish (translation done by Stevan Vanderwerf)
+* Support for Unicode 10.0
+
+## 9.0.522 stable
```

Comment:
This might take some investigative reporting, but shouldn't the format include release dates?
(the alpha on top won't need it)

```
## yyyy-mm-dd major.minor.build tier
```

Reply:
That would be nice, wouldn't it?  If only our existing release notes included them.

Reply:
It's not a requirement for 'legacy' (pre 10.0) release notes, since some of that data may well be lost to time.  It validates properly with the shell script resources.